src/assets/style/components/merkle_graph.scss

```diff
@@ -7,6 +7,13 @@
         margin-top: 20px;
         border-radius: 7.5px;
 
+        animation: growMerkle .5s;
+
+        @keyframes growMerkle {
+            0% {max-height: 0px; padding-top: 0px; padding-bottom: 0px}
+            100% {max-height: 2400px;}
+        }
+
```

Comment:
734ab612809b11cd44e370437487ffb765b657ac: Let's remove this commit. Excessive animations make the code more complex and are prone to break. Windows popping up upon a button click are enough.